.github/workflows/release.yaml

```diff
@@ -21,8 +21,10 @@ jobs:
       fail-fast: false
       matrix:
         job:
-          - { os: ubuntu-18.04 , target: x86_64-unknown-linux-gnu    }
-          - { os: macos-10.15  , target: x86_64-apple-darwin         }
+          - { os: ubuntu-20.04 , target: x86_64-unknown-linux-gnu    }
+          - { os: ubuntu-20.04 , target: aarch64-unknown-linux-gnu    }
+          - { os: macos-12     , target: x86_64-apple-darwin         }
+          - { os: macos-12     , target: aarch64-apple-darwin         }
```

Comment:
if I weren't lazy and sleep-deprived right now, I could probably make a fancy workflow that makes a fat binary containing both architecture... but I'm lazy and sleep-deprived right now.

Reply:
Hahaha, either way it's much better off now so thank you!